### ISSUE TITLE
Error not retrieving number of followers

### ISSUE THREAD

**Issue by arc9531** [instabot-py/instabot.py#2576]
I let the bot run for a bit then I start getting an error that says:
InstaBot - WARNING - Could not retrieve number of followers of user

**Comment by nickgryg** [comment]
It seems you were banned by Instagram. Please update `instabot-py` to version `0.7.7`. It should help you with the further investigation.

**Comment by arc9531** [comment]
I updated it but whenever I run the `instabot-py` command, It says that it is outdated! How can I fix this?

**Comment by nickgryg** [comment]
Upgrade can be done with the following command:
```
python3 -m pip install instabot-py --no-cache-dir --upgrade
```

**Comment by arc9531** [comment]
I've done that and still get a notification that says `Newer version is available: 0.7.8. The current version is: 0.7.1.
To update instabot-py, please perform:
    python3 -m pip install instabot-py --upgrade --no-cache-dir`

**Comment by arc9531** [comment]
?

**Comment by nickgryg** [comment]
Original issue has been fixed in version `0.7.8`.
As far as I see another issue with upgrade was created in #2605 

Closing this one. Cheers.
